ldes-fragmentisers/ldes-fragmentisers-geospatial/src/main/java/be/vlaanderen/informatievlaanderen/ldes/fragmentisers/geospatial/connected/relations/GeospatialRelationsAttributer.java

```diff
@@ -34,7 +34,7 @@ private void addRelationToTargetFragment(LdesFragment sourceFragment, LdesFragme
         String targetWKT = getWKT(targetFragment);
 
         TreeRelation relationToTargetFragment = new TreeRelation(GEOSPARQL_AS_WKT, targetFragment.getFragmentId(),
-                targetWKT, TREE_GEOSPATIALLY_CONTAINS_RELATION);
+                WGS_84+" "+ targetWKT, WKT_DATA_TYPE, TREE_GEOSPATIALLY_CONTAINS_RELATION);
```

Comment:
still convinced that the "GEOSPARQL_AS_WKT" is a value that should come from your config, not hardcoded.